platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -782,6 +784,9 @@ fn build_solana_package(
     if config.remap_cwd && !config.debug {
         target_rustflags = Cow::Owned(format!("{} -Zremap-cwd-prefix=", &target_rustflags));
     }
+    if !config.optimize_cu {
+        target_rustflags = Cow::Owned(format!("{} -C opt-level=s", &target_rustflags));
+    }
```

Comment:
I'm also open to leaving the compiler flag behind a new argument (the opposite of what I did here).

Reply:
I'm going to try the opposite, since the CU increase is so large that some programs in the downstream tests stopped working.